site-manage/site-manage-impl/impl/src/java/org/sakaiproject/sitemanage/impl/SiteManageServiceImpl.java

```diff
@@ -76,6 +79,9 @@
 @Slf4j
 public class SiteManageServiceImpl implements SiteManageService {
 
+    // Duplicated from basiclti/tsugi-util/src/java/org/tsugi/lti13/LTICustomVars.java
+    public static final String CONTEXT_ID_HISTORY = "Context.id.history";
```

Comment:
I am happy to add the import from LTICustomVars and add a dependency to the pom.xml if you think that would be a better approach.  I guess that in the long run that SiteManage will learn more and more about LTI so that is OK.  WDYT?